File: lab/python/binarios/decimal.py
def retornar_binario(numero):
    lista_restos = []
    while numero != 0:
        numero, resto = numero // 2, numero % 2
        lista_restos.append(resto)

    resultado = ""
    for i in range(len(lista_restos)-1, -1, -1):
        resultado += str(lista_restos[i])
    return resultado


def retornar_binario2(numero):
    resultado = ""
    while numero != 0:
        temp = numero % 2
        resto = temp
        numero //= 2
        resultado += str(resto)
    return resultado[::-1]


def retornar_binario3(numero):
    lista_restos = []
    while numero != 0:
        numero, resto = numero // 2, numero % 2
        lista_restos.append(resto)

    return ''.join(str(bit) for bit in reversed(lista_restos))


print(retornar_binario(10))
print(retornar_binario2(10))
print(retornar_binario3(10))

File: lab/javascript/binarios/decimal.js
function retornar_binario(numero){
    let lista_restos = [];

    while (numero != 0){
        let temp = numero % 2;
        let resto = temp;
        numero = parseInt(numero /= 2);
        lista_restos.push(resto);
    }

    resultado = "";
    for (let i = lista_restos.length -1; i > -1; i--){
        resultado += String(lista_restos[i]);
    }
    return resultado;
}

console.log(retornar_binario(10))
